src/lib/pipeline.ml

```diff
@@ -316,16 +331,21 @@ let rec to_file_prefix:
       sprintf "%s-star-aligned" (to_file_prefix ?is sample)
     | Hisat (sample) ->
       sprintf "%s-hisat-aligned" (to_file_prefix ?is sample)
-    | Stringtie (sample) ->
-      sprintf "%s-stringtie" (to_file_prefix ?is sample)
+    | Stringtie (use_reference_gtf, sample) ->
+      sprintf "%s-%sstringtie"
+        (to_file_prefix ?is sample)
+        (match use_reference_gtf with
+        | `Yes -> "yesgtf"
```

Comment:
I prefer the 3-val variant; it's more readable and gives more power.

My problem is choosing the default value; I picked `` `If_available`` because it's the one *I* would prefer but maybe forcing the failure when there is no GTF is a better default (it would be a "Heads Up, something's wrong" case)